frontend/src/pages/modelRegistry/ModelRegistryCoreLoader.tsx

```diff
@@ -89,18 +114,16 @@ const ModelRegistryCoreLoader: React.FC<ModelRegistryCoreLoaderProps> =
     return (
       <ApplicationsPage
         title={
-          <TitleWithIcon
-            title="Registered models"
-            objectType={ProjectObjectType.registeredModels}
-          />
+          <TitleWithIcon title="Model registry" objectType={ProjectObjectType.registeredModels} />
```

Comment:
I didn't find `Model registry` in the title in the mocks(the ones attached to the JIRA). Let me know if I'm missing something. If `Model registry` is the title, please update the screenshots in the PR description. I saw a new description "View and manage all of your registered models. Registering models to model registry allows you to manage their content, metadata, versions, and user access settings.", is this description for all pages with `Registered models` as title? This is perhaps a question for @yih-wang I found another page with `Registered Models` and a description - "View and manage your registered models.". If this description is for all pages, please update it accordingly.

Reply:
That copy change is a part of this separate task: https://issues.redhat.com/browse/RHOAIENG-6574

Reply:
I ended up updating this specific text anyways for the sake of cypress tests passing.